File: MPI/mpicheck/tests/functions/3.0/MPI_Win_attach.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       TYPE(INTEGER) var_1(10)
       INTEGER(KIND=MPI_ADDRESS_KIND) var_2
       INTEGER var_3
        call mpi_win_attach(var_0, var_1, var_2, var_3)
       call pmpi_win_attach(var_0, var_1, var_2, var_3)
        end program main
    

File: MPI/mpicheck/tests/functions/3.0/MPI_Win_attach.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Win var_0;
    void *var_1;
    MPI_Aint var_2;
    int ret;
    /* calls */
    ret = MPI_Win_attach(var_0, var_1, var_2);
    ret = PMPI_Win_attach(var_0, var_1, var_2);
    return 0;
}
